test/e2e/performanceprofile/functests/utils/pods/pods.go

```diff
@@ -149,7 +149,7 @@ func ExecCommandOnPod(c *kubernetes.Clientset, pod *corev1.Pod, containerName st
 			Stdin:     true,
 			Stdout:    true,
 			Stderr:    true,
-			TTY:       true,
+			TTY:       false,
```

Comment:
for historical record: in the review of the main branch PR we found the likely cause why this fix is helping